tests/unit/orchestrate/flow/flow-orchestrate/test_flow_complex_topology.py

```diff
@@ -36,15 +37,14 @@ def test_flow_external_executor_with_gateway():
     def serve_exec(**kwargs):
         FooExec.serve(**kwargs)
 
-    e = threading.Event()
-    t = threading.Thread(
+    e = multiprocessing.Event()
+    t = multiprocessing.Process(
         name='serve-exec',
         target=serve_exec,
-        kwargs={'port_expose': external_gateway_port, 'stop_event': e},
-        daemon=True,
+        kwargs={'port': external_gateway_port, 'stop_event': e},
```

Comment:
daemon=True proved to prevent CI hanging when we do multiprocessing.
are we sure we want to remove it ?
@samsja fyi

Reply:
if we do not remove, we cannot start Processes in it. And in multithreads, no signal handling is provided.